File: Web/Macabre908_AbhishekDubey_2125cse1175_2/FunctionAndArrays/spanOfArray.java
import java.util.*;

public class spanOfArray {
    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int[] arr = new int[n];
        for(int i = 0; i < n; i++){
            arr[i] = sc.nextInt();
        }
        
        int max = arr[0];
        int min = arr[0];
        
        for(int i = 1; i < n; i++){
            if(arr[i] > max){
                max = arr[i];
            } 
            if(arr[i] < min){
                min = arr[i];
            }
        }
        
        int span = max - min;
        System.out.println(span);
        sc.close();
     }
    
}

File: ML/KeplerInCoding_ShreyaSharma_2125cse1170_2/FunctionAndArrays/SpanOfArray.cpp
#include <cstdio>
int main()
{
    int n; 
    scanf("%d",&n);
    int a[n];
    for(int i=0; i<n; i++)
    {
        scanf("%d", &a[i]);
    }
    int max= a[0];
    int min= a[0];

    for(int i=1; i<n; i++)
    {
        if(min>a[i])
        {
            min=a[i];
        }
        if(max<a[i])
        {
            max=a[i];
        }
    }
    printf("%d", max-min);
    return 0;
}